official/nlp/keras_nlp/layers/transformer_encoder_block.py

```diff
@@ -148,7 +145,6 @@ def build(self, input_shape):
         kernel_initializer=self._attention_initializer,
         name="self_attention",
         **common_kwargs)
-    self._attention_dropout = tf.keras.layers.Dropout(rate=self._output_dropout)
```

Comment:
Please keep "self._attention_dropout":
self._attention_dropout_layer = tf.keras.layers.Dropout(rate=self._output_dropout)

The dropout in MultiHeadAttention is different to this attention dropout.

Reply:
Done. The weird thing is that it receives `self._output_dropout`, the same as a `self._output_dropout_layer`